.github/workflows/deploy-non-prod-web.yml

```
@@ -10,16 +10,28 @@ on:
         default: 'QA'
         type: choice
         options:
-          - QA
+          - USQA
           - EUQA
           - USPROD
+          - EUPROD
+      tag:
```

```suggestion
      tag:
```

Should this be called `tag` if we don't currently support tags?

---

Does the workflow really not support tags? I was under the impression all the deploy workflows do.